models/yang/sonic/sonic-vrf.yang

```diff
@@ -0,0 +1,45 @@
+module sonic-vrf {
+    namespace "http://github.com/Azure/sonic-vrf";
+    prefix vrf;
+
+    import sonic-common {
+        prefix scommon;
+    }
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description
+        "SONIC VRF";
+
+    revision 2019-10-30 {
+        description
+            "Initial revision.";
+    }
+
+    container sonic-vrf {
+        container VRF {
```

Comment:
It should be VRF_TABLE not VRF, please correct it, please refer VRF HLD.

Reply:
The config DB name for vrf table is "VRF" per HLD.